src/test/java/teammates/test/cases/ui/browsertests/AdminSearchPageUiTest.java

```diff
@@ -68,6 +78,94 @@ private AdminSearchPage getAdminSearchPage(String instructorId) {
         return loginAdminToPage(browser, commentsPageUrl, AdminSearchPage.class);
     }
     
+    private boolean isPageTitleCorrect() {
+        return searchPage.getPageTitle().equals("Admin Search");
+    }
+    
+    private boolean isSearchPanelVisible() {
+        return searchPage.isElementPresent(By.id("filterQuery"))
+            && searchPage.isElementPresent(By.id("searchButton"));
+    }
+    
+    private boolean isEmptyKeyErrorMessageShown() {
+        String statusMessage = searchPage.getStatus();
+        
+        return statusMessage.equals("Search key cannot be empty");
+    }
+    
+    /**
+     * This method only checks if the search data tables are displayed correctly
+     * i.e, table headers are correct, and appropriate message is displayed if no
+     * search data is present.
+     * It does not test for the table content
+     */
+    private boolean isSearchDataDisplayCorrect() {
+        if (searchPage.isElementPresent(By.className("table"))) {
+            int numSearchDataTables = browser.driver.findElements(By.className("table")).size();
+            for (int i = 0 ; i < numSearchDataTables ; i++) {
+                if (!isSearchTableHeaderCorrect(i)) {
+                    return false;
+                }
+            }
+            return true;
+        } else {     
+            String statusMessage = searchPage.getStatus();
+            return statusMessage.equals("No result found, please try again");
+        }
+        
+    }
+    
+    private boolean isSearchTableHeaderCorrect(int tableNum) {
+        List<String> expectedSearchTableHeaders;
+        List<String> actualSessionTableHeaders;       
+
+        int numColumns = searchPage.getNumberOfColumnsFromDataTable(tableNum);
+        
+        switch (searchPage.getDataTableId(tableNum)) {
+        // Instructor table
+        case "search_table_instructor":
+            if (numColumns != 5) {
```

Comment:
You could extract this to a constant in this class. Perhaps something like `INSTRUCTOR_SEARCH_TABLE_NUM_COLUMNS`